.github/workflows/main.yaml

```diff
@@ -4,83 +4,64 @@ on:
   push:
     branches:
       - maintenance-3.0.x
-  pull_request:
+      - ELEMENTS-1459-pr-check-split
```

Comment:
do we need this?

Reply:
Either way, we don't need this, and this should be completely removed.